site/content/internal/sustained-engineering.md

```diff
@@ -81,9 +81,19 @@ SET attends all triage meetings. During triage, SET should:
 
 During non-release weeks, triage is held twice a week. Near the end of the release cycle, triage is held daily.
 
-## Support Responses
+## Customer and Pre-Sales Support Escalation Process
 
-Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. To help us do this better, SET members should use the following responses when receiving issue escalations from the customer support team:
+Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. When the pre-sales and customer support teams require engineering support, they should:
+
+1. Post the issue in [Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel. If the message contains confidential information about a customer, then post in [Customer Support](https://community.mattermost.com/private-core/channels/community) channel in private Staff team.
+2. At-mention SET team lead, identified as the first person in the [Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel header.
+3. If no prompt solution is presented, create a ticket in [Mattermost Jira project](https://mattermost.atlassian.net/) and assign "Sustained Engineering" as the **Mattermost Team**. The ticket will be reviewed by a SET team member during Tuesday or Thursday triage.
```

Comment:
Reviewed by the SET team not a member. Also I would recommend that the person that file the ticket to be present at the meeting before the ticket gets triaged to be able to answer any questions that cannot be answered with the info on the ticket

Reply:
Updated to just `SET team`.

I think any questions that need to be answered regarding the ticket should be done through jira/MM. If SET team thinks they need a high bandwidth conversation with the support team, I suggest that SET engages the support team on the side, as opposed to always getting them to join triage meetings. If something is high priority they'll automatically bring it up during triage meetings.

Thoughts?